src/components/units-list/units-list.tsx

```diff
@@ -15,6 +16,7 @@ export interface IUnitsListProps {
   className?: string
   isDark?: boolean
   withCharts?: boolean
+  downloadedData: number
```

Comment:
Without looking there it's not obvious that downloadedData is just a primitive, not a structure.

Reply:
Changed